packages/caver-account/src/index.js

```diff
@@ -0,0 +1,209 @@
+     */
+    static createWithAccountKeyFail(address) {
+        return new Account(address, new AccountKeyFail())
+    }
+
+    /**
+     * creates an Account instance which has AccountKeyWeightedMultiSig as an accountKey
+     *
+     * @param {string} address The address of Account.
+     * @param {Array} publicKeyArray The array that includes multiple public key strings.
+     * @param {Object} options The object that includes threshold and weight array.
+     * @return {Account}
+     */
+    static createWithAccountKeyWeightedMultiSig(address, publicKeyArray, options) {
+        if (options === undefined)
+            throw new Error(`options is undefined. To create Account instance with AccountKeyWeightedMultiSig options should be defined.`)
+        return new Account(address, AccountKeyWeightedMultiSig.fromPublicKeysAndOptions(publicKeyArray, options))
+    }
+
+    /**
+     * creates an Account instance which has AccountKeyRoleBased as an accountKey
+     *
+     * @param {string} address The address of Account.
+     * @param {Array} roledBasedPublicKeyArray A two-dimensional array containing arrays of public key strings for each role.
+     * @param {Array} options An array that contains objects with threshold and weight array defined for each role.
+     * @return {Account}
+     */
+    static createWithAccountKeyRoleBased(address, roledBasedPublicKeyArray, options) {
+        if (options === undefined)
+            throw new Error(`options is undefined. To create Account instance with AccountKeyRoleBased options should be defined.`)
+        return new Account(address, AccountKeyRoleBased.fromRoleBasedPublicKeysAndOptions(roledBasedPublicKeyArray, options))
+    }
+
+    /**
+     * Create an account.
+     * @param {string} address - The address of account.
+     * @param {AccountKeyLegacy|AccountKeyPublic|AccountKeyFail|AccountKeyWeightedMultiSig|AccountKeyRoleBased} accountKey - The accountKey of account.
+     */
+    constructor(address, accountKey) {
+        if (!utils.isAddress(address)) throw new Error(`Invalid address : ${address}`)
+        if (!isAccountKeyInstance(accountKey))
+            throw new Error(
+                `Invalid accountKey. accountKey should be an instance of AccountKeyLegacy, AccountKeyPublic, AccountKeyFail, AccountKeyWeightedMultiSig or AccountKeyRoleBased`
+            )
+
+        this._address = utils.addHexPrefix(address)
+        this._accountKey = accountKey
+    }
+
+    /**
+     * @type {string}
+     */
+    get address() {
+        return this._address
+    }
+
+    set address(addressInput) {
+        if (!utils.isAddress(addressInput)) throw new Error(`Invalid address : ${addressInput}`)
+
+        this._address = utils.addHexPrefix(addressInput)
+    }
+
+    /**
+     * @type {AccountKeyLegacy|AccountKeyPublic|AccountKeyFail|AccountKeyWeightedMultiSig|AccountKeyRoleBased}
+     */
+    get accountKey() {
+        return this._accountKey
+    }
+
+    set accountKey(accountKey) {
```

Comment:
No comments for this function?

Reply:
I initially added descriptions for both get/set, but when I experimented with jsDoc, I found the problem that it would duplicate the same descriptions.
That's why I think only one explanation is better for get/set functions for class member variables.
Right now, even if caver-js doesn't docs automation, it can reduce additional work in the future.